### DISCUSSION TITLE
Why is the commit message being converted to lowercase?

### DISCUSSION THREAD

**Discussion by yannickkirschen** [renovatebot/renovate#31961] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab, latest version of Renovate (renovate/renovate:latest)

### Please tell us more about your question or problem

Hello there,

we extended the configuration to append a Jira issue to all commits, to satisfy internal regulations. Our config looks like this:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "cloneSubmodules": true,
  "git-submodules": {
    "enabled": true
  },
  "commitMessageSuffix": "(XXX-0000)"
}
```

We noticed, that the commit message is being converted to lowercase. So we expect a commit message like this: `chore(deps): update xyz to 1.2.3 (XXX-0000)`, but renovate commits: `chore(deps): update xyz to 1.2.3 (xxx-0000)`. This way, the connection GitLab -> Jira does not work.

We found that https://github.com/renovatebot/renovate/issues/12062 addresses this issue and has been closed. But we still have the same behaviour, although we use the latest image version.

Does anyone know whats going wrong here?

Cheers!
Yannick

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Related: https://docs.renovatebot.com/configuration-options/#commitmessagelowercase

**Reply by yannickkirschen** [reply]
Thanks, this helped out!
